examples/experimental/i3s-17-and-debug/bounding-volume-layer.js

```diff
@@ -78,49 +85,37 @@ export default class BoundingVolumeLayer extends CompositeLayer {
       ...attributes.POSITION,
       value: new Float32Array(attributes.POSITION.value)
     };
+
```

Comment:
Add some comments explaining what you are doing? If you are just translating the sphere, isn't that better done with `modelMatrix`?

Reply:
We are tried to play with model matrix while creating mbs geometry and it doesn't work. Now we just pass `modelMatrix` as prop to the `MeshLayer` with generated mbs geometry. At this line we just do deep copy of `SphereGeometry` mesh to override it when we add center to each vertex.
Do you have better solution for this?